bin/wasabi.sh

```diff
@@ -282,6 +282,8 @@ for command in ${@:$OPTIND}; do
     start) command="start:cassandra,mysql,wasabi";&
     start:*) commands=$(echo ${command} | cut -d ':' -f 2)
       (IFS=','; for command in ${commands}; do start ${command}; done);;
+    test:*) commands=$(echo ${command} | cut -d ':' -f 2)
```

Comment:
Added it to the usage with [7092306](https://github.com/intuit/wasabi/pull/74/commits/7092306ea567b3866c8652cf9e5b6f2cf071565f)